ramp-database/ramp_database/cli.py

```diff
@@ -189,6 +191,52 @@ def add_submission(config, event, team, submission, path):
                                          path)
 
 
+@main.command()
+@click.option("--config", default='config.yml', show_default=True,
+              help='Configuration file YAML format containing the database '
+              'information')
+@click.option("--config_event", required=True,
+              help="""Path to configuration file YAML format
+              containing the database information, eg config.yml""")
+@click.option('--event-name', required=True,
+              help='Name of the event which will be removed.')
+@click.option('--from-disk', is_flag=True, default=False,
+              help="""If True the event and all the related submissions
+                      will be removed from the database and from the disk;
+                      if set to False they will only be removed from
+                      the database""")
+def delete_event(config, config_event, event_name, from_disk):
+    """Delete event."""
+    config = read_config(config)
+
+    # check if the event_config dir exists
+    if not os.path.isfile(config_event) and from_disk:
+        print('cannot find '+config_event)
+        from_disk = False
```

Comment:
Actually we should be raising an error to not go further. The echo something like

```python
click.echo(
    "The event config file is not existing. This file is required for"
    "deleting files from the disks. Either remove the flag --from-disk"
    "or give the correct path to the event config file.",
    err=True
)
```